charts/openshift-metering/templates/openshift-reporting/report-queries/persistentvolumeclaim-usage.yaml

```diff
@@ -205,7 +205,7 @@ spec:
       timestamp '{| default .Report.ReportingStart .Report.Inputs.ReportingStart| prestoTimestamp |}' AS period_start,
       timestamp '{| default .Report.ReportingEnd .Report.Inputs.ReportingEnd | prestoTimestamp |}' AS period_end,
       namespace,
-      sum(persistentvolumeclaim_usage_bytes)
+      avg(persistentvolumeclaim_usage_bytes)
```

Comment:
Just picking this back up now, but I realize that using sum here is probably the most reasonable way to calculate usage per-PVC-per-namespace, but rather the value of the `persistentvolumeclaim_usage_bytes` is likely way off in the way we calculate and expose that column. We likely should be summing the averaged value for an individual PVC here instead of summing all of the metric samples we've gathered between a datetime and timestamp range, which would help explain why we were seeing orders-of-magnitude results when running this ReportQuery custom resource.